app/assets/src/components/common/MetadataManualInput.jsx

```diff
@@ -90,12 +86,6 @@ class MetadataManualInput extends React.Component {
     ];
   };
 
-  handleHostGenomeChange = id => {
```

Comment:
I'm assuming you're going to remove this in https://github.com/chanzuckerberg/idseq-web/pull/1988